Mintlify Chat Blog post (#48)
mintlify/mintlify.com: src/pages/blog/chat/index.mdx

import { ArticleLayout } from '@/components/ArticleLayout'

export const meta = {
  title: 'Mintlify Chat - ChatGPT trained on your documentation',
  description:

>     'Why we upgraded the answer discovery experience and how we did it with Inkeep',

Reviewer: ```suggestion
    'How we upgraded the information discovery experience with Inkeep',
```

Reviewer: the wording of this description is weird imo
